guides/throttle.md

```diff
@@ -5,9 +5,14 @@ See `amoc_throttle`
 ## Overview
 
 Amoc throttle is a module that allows limiting the number of users' actions per given interval, no matter how many users there are in a test.
-It works in both local and distributed environments, allows for dynamic rate changes during a test and exposes metrics which show the number of requests and executions.
+It works in both local and distributed environments, allows for dynamic rate changes during a test and exposes telemetry events showing the number of requests and executions.
+
+Amoc throttle allows to:
+
+- Setting the execution `Rate` per `Interval`
+- Limiting the number of parallel executions when `Rate` is set to `infinity`
```

Comment:
`Rate` is supposed to be the number of parrallel executions if `Interval` is 0

Reply:
@NelsonVides, any comment here?

Reply:
Fixed in a newer version of the code.